tests/ctl/api/test_admin.py

```diff
@@ -26,12 +26,13 @@ def test_db_reset_dev_mode_disabled(
     test_config: FidesConfig,
     test_config_dev_mode_disabled: FidesConfig,  # temporarily switches off config.dev_mode
     test_client: TestClient,
+    loguru_caplog,
 ) -> None:
-    with pytest.raises(
```

Comment:
Since we're now catching exceptions instead of letting them exit the process, this still gets logged but doesn't blow things up (which the test expected)